ramls/oaipmh/oaipmhinstanceids.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for oai-pmh updated instances ids view",
+  "type": "object",
+  "properties": {
+    "instanceId": {
+      "description": "OAI-PMH updated instances ids",
+      "type": "string"
```

Comment:
Done

Reply:
Ah, good point. Do you mean that this schema describes response only?

Usually we specify format to better explain API, so API consumers now what data format is expected for a property.

Reply:
Indeed, the primary intent of the API definition is to describe to the potential clients. That FOLIO sometimes also uses it to drive behaviour is a secondary concern from my perspective.

Reply:
In the last changes I added for all newly introduced schemas reference to uuid defirnition. But taking into account that uuid definition located in different path and to allow relative reference I was updated uuid definiion itself in term of valid schema definition. Now it can be referenced from everywhere by relative path specifying.